### PR TITLE
Add GitHub Actions workflows

### PR DESCRIPTION
**Pull request by Turbo87**
... because CircleCI seems to be failing for no apparent reason 🤷‍♂️

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      - uses: actions/setup-node@v1
+        with:
+          registry-url: 'https://registry.npmjs.org'
+
+      - name: auto-dist-tag
+        run: npx auto-dist-tag@1 --write
+
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
```
Comment (rwjblue) [inline .github/workflows/release.yml:24]: Is this token setup already?

Comment (Turbo87) [reply .github/workflows/release.yml:24]: nope, not yet

Comment (kategengler) [reply .github/workflows/release.yml:24]: Added the token

**Comment by kategengler** [comment]
Thank you!

**Pull request merged**